packages/mdc-textfield/README.md

```diff
@@ -176,6 +176,32 @@ UX for client-side form field validation.
 </p>
 ```
 
+### Leading and Trailing Icons
```

Comment:
The [spec](https://material.io/guidelines/components/text-fields.html#text-fields-input-types) shows _text_ prefixes and suffixes in addition to icons, and the issues linked to in this PR (#677 and #678) seem to be talking about the text versions.

Are we planning to support the text prefixes/suffixes in a separate PR?

Reply:
The result of the most recent meeting I had was to work against the [M2 spec](https://spec.googleplex.com/m2/components/text-fields.html#) which makes no mention of text prefixes/suffixes

Reply:
Oops! You're right, my bad 😝